.github/workflows/node-flow-deploy-release-artifact.yaml

```diff
@@ -154,12 +163,17 @@ jobs:
           token: ${{ secrets.GH_ACCESS_TOKEN }}
           inputs: '{ "ref": "${{ inputs.ref }}" }'
 
-#      - name: Trigger ZXF Deploy Integration
-#        if: ${{ needs.release-branch.result == 'success' }}
-#        uses: step-security/workflow-dispatch@4d1049025980f72b1327cbfdeecb07fe7a20f577 # v1.2.4
-#        with:
-#          workflow: .github/workflows/node-zxf-deploy-integration.yaml
-#          repo: hashgraph/hedera-services # ensure we are executing in the hashgraph org
-#          ref: develop # ensure we are always using the workflow definition from the develop branch
-#          token: ${{ secrets.GH_ACCESS_TOKEN }}
-#          inputs: '{ "event": "${{ inputs.event }}" }'
+      - name: Trigger ZXF Deploy Integration
+        if: ${{ needs.release-branch.result == 'success' }}
+        uses: step-security/workflow-dispatch@4d1049025980f72b1327cbfdeecb07fe7a20f577 # v1.2.4
+        with:
+          workflow: .github/workflows/node-zxf-deploy-integration.yaml
+          repo: hashgraph/hedera-services # ensure we are executing in the hashgraph org
+          ref: develop # ensure we are always using the workflow definition from the develop branch
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          inputs: '{
+            "ref": "${{ inputs.ref }}",
+            "author": "${{ inputs.author }}",
+            "message": "${{ inputs.msg }}",
+            "headSha": "${{ inputs.sha }}"
```

Comment:
```suggestion
            "author": "${{ inputs.author }}",
            "msg": "${{ inputs.msg }}",
            "sha": "${{ inputs.sha }}"
```

Reply:
Saw this literally right after I committed. Fixed.